custom-login/src/main.js

```diff
@@ -22,12 +22,14 @@ import sampleConfig from '@/config'
 
 const oktaAuth = new OktaAuth(sampleConfig.oidc)
 
+export const emitter = oktaAuth.emitter
+
 createApp(App)
   .use(router)
   .use(OktaVue, {
     oktaAuth,
     onAuthRequired: () => {
-      router.push('/login')
+      emitter.emit('_onAuthRequired', oktaAuth);
```

Comment:
Why are we using the emitter this way? Can we just call a function to show the modal dialog directly?

Reply:
Got rid of emitter